### ISSUE TITLE
Missing-image icon in Sound panel

### ISSUE THREAD

**Issue by bertob** [sonnyp/Tangram#191]
I'm getting these icons for Tangram audio sources in Settings:

![image](https://user-images.githubusercontent.com/1908896/166107106-50df93e3-b13e-4689-87d1-f68b9760add4.png)

**Comment by sonnyp** [comment]
I did some digging and compared with an app that doesn't present the issue - Shortwave.

Both icons are at the same location

```
~/.local/share/flatpak/exports/share/icons/hicolor/symbolic/apps/de.haeckerfelix.Shortwave-symbolic.svg
~/.local/share/flatpak/exports/share/icons/hicolor/symbolic/apps/re.sonny.Tangram-symbolic.svg
```

However in Settings - it uses the icon-name `de.haeckerfelix.Shortwave-symbolic` for Shortwave and `audio-symbolic` for Tangram.

Looks like there are 2 issues.

Settings uses `audio-symbolic` as fallback but the icon doesn't exist - I filed https://gitlab.gnome.org/GNOME/gnome-control-center/-/issues/1910

GNOME can't figure out which app (Tangram) is outputting audio.

I can reproduce with Tangram, Tangram, Devel and Epiphany
I can't reproduce with Epiphany Devel
(all flatpaks)
